Sources/PackageCollections/Model/Package.swift

```diff
@@ -142,6 +131,22 @@ extension PackageCollectionsModel.Package {
 
         /// The package version's license
         public let license: PackageCollectionsModel.License?
+
+        public struct Manifest: Equatable, Codable {
+            /// The Swift tools version specified in `Package.swift`.
+            public let toolsVersion: ToolsVersion
```

Comment:
Note that there are two `ToolsVersion` here.

@abertelrud @neonichu 

The "key" `ToolsVersion` is the one specified in `Package.swift` (if default), otherwise it's the one specified in the filename.

`Manifest.toolsVersion` is what's specified in the manifest for all cases. It's more convenient to have `Manifest.toolsVersion`, but I don't know if we should assign the manifest's tools version to it or the "key" `ToolsVersion`. As we discussed before, the two can be different.

Reply:
Discussed with team, the code should validate that "key" tools version should match value `Manifest.toolsVersion`. https://github.com/apple/swift-package-manager/pull/3226/commits/d588f88a82934254fafccb20d56b25cff3cbafff